src/style.css

```diff
@@ -1,3 +1,64 @@
-body {
+.body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 22.5px 0 ;
+
+}
+
+.nav {
+  margin: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  padding: 22.5px 0;
+  box-sizing: border-box;
+  position: relative;
+
+}
+
+.nav__item :hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
```

Comment:
you repeating yourself at least three times. so better to move this to some variable